components/workspace-rollout-job/pkg/rollout/metrics.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package rollout
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
```

Comment:
It's really awesome that you want to be able to monitor the rollout job!

As we discussed in slack, metrics might not be the best approach here. What do you think about deleting all code related to metrics and relying entirely on logs?